managed/ui/src/components/universes/UniverseDetail/UniverseDetail.js

```diff
@@ -620,6 +622,28 @@ class UniverseDetail extends Component {
                         />
                       )}
 
+                      <YBMenuItem
```

Comment:
We need to disable this for the `Pause Universe` state. So please check that before merging this change.

Reply:
Run `prettier` as well.